package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "karma start karma.conf.js"
```

Comment:
Might want to revert this change since we want to run karma tests.

Reply:
Also, there are a lot of deprecation warnings in the build console. Lets bump up the packages and try to use the latest ones (if possible). Would give us some speed up in terms of loading for sure.